@xen-orchestra/fs/src/local.js

```diff
@@ -48,6 +48,9 @@ export default class LocalHandler extends RemoteHandlerAbstract {
   }
 
   _getInfo() {
+    // Resolves with an object with the following properties:
+    // filesystem, type, size, used, available, capacity and mountpoint.
+    // size, used, available and capacity may be `NaN`
```

Comment:
Or should we clear the `NaN` direclty here?

Reply:
That would work. I think it would be weird to "hide" the errors (even if they're logged) directly inside `formatSize`.